{"status": "broken", "name": "Matching variable length pattern with property predicate", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "skip"}, {"name": "feature", "value": "MatchAcceptance2"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython2"}], "stop": 1578486074332, "historyId": "0e741f28b05fb3e455d37a78c46fb543", "start": 1578486074211, "steps": [{"status": "passed", "start": 1578486074212, "stop": 1578486074322, "name": "Given an empty graph"}, {"status": "passed", "start": 1578486074323, "stop": 1578486074327, "name": "And having executed", "attachments": [{"source": "d685c80c-1def-44a3-9720-09ce9140f9bc-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "passed", "start": 1578486074328, "stop": 1578486074330, "name": "When executing query", "attachments": [{"source": "d6ebefe9-c01f-45d9-af8e-8919a9132627-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "broken", "attachments": [{"source": "1e7b1127-a46c-4286-9894-fa39974c7b2a-attachment.csv", "type": "text/csv", "name": ".table"}], "stop": 1578486074332, "start": 1578486074330, "statusDetails": {"message": "ResponseError: RedisGraph does not currently support filters on variable-length paths.\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 84, in step_impl\n    raise exception\n"}, "name": "Then the result should be"}, {"status": "skipped", "start": 1578486074332, "stop": 1578486074332, "name": "And no side effects"}], "statusDetails": {"message": "ResponseError: RedisGraph does not currently support filters on variable-length paths.\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 84, in step_impl\n    raise exception\n"}, "uuid": "f06259b1-10ca-4e55-9dfa-3ec902bdbd8e"}